integration-tests/features/steps/gemini.py

```diff
@@ -77,6 +77,18 @@ def access_gemini_url(context, endpoint, token="without"):
     context.response = requests.get(url, headers=headers)
 
 
+@when('I access the {endpoint} endpoint of Gemini service for {parameter} report {list}')
+@when('I access the {endpoint} endpoint of Gemini service for {parameter} report')
+def access_stacks_report_list(context, endpoint, parameter='', list=''):
```

Comment:
It would be better not to use `list` as variable name, because it shadows `list` constructor.